lib/views/player/widgets/player_buttons/play_pause_button_widget.dart

```diff
@@ -9,7 +9,7 @@ class PlayPauseButtonWidget extends ConsumerWidget {
   @override
   Widget build(BuildContext context, WidgetRef ref) {
     final provider = ref.watch(audioPlayerNotifierProvider);
-    ref.watch(audioPlayPauseProvider);
+    // ref.watch(audioPlayPauseProvider);
```

Comment:
?

Reply:
@osamaasifoak Once this is fixed feel free to merge. Cheers!

Reply:
comment removed